demo/cluster/aws-standalone-regional.yaml

```
@@ -9,12 +9,15 @@ metadata:
     k0rdent.mirantis.com/kof-cluster-role: regional
 spec:
   credential: aws-cluster-identity-cred
-  template: aws-standalone-cp-0-1-5
+  template: aws-standalone-cp-0-1-6
   config:
     clusterAnnotations:
       k0rdent.mirantis.com/kof-regional-domain: aws-ue2.kof.example.com
       k0rdent.mirantis.com/kof-cert-email: mail@example.com
 
+      # Any custom values, e.g. https://docs.victoriametrics.com/helm/victorialogs-single/index.html#parameters
+      k0rdent.mirantis.com/kof-storage-values: '{"victoria-logs-single": {"server": {"replicaCount": 2}}}'
```

Why JSON? Not Yaml like this:
```suggestion
      k0rdent.mirantis.com/kof-storage-values: |
        victoria-logs-single:
          server:
            replicaCount: 2
```

---

Thanks, done: https://github.com/k0rdent/kof/pull/169/commits/28da3bcac7ea42d8d4669f1922e1e9647ab3e380

Also verified it really creates two pods of `kof-storage-victoria-logs-single-server`, see the issue description.